WixInstaller/XenCenter.l10n.diff

```diff
@@ -174,8 +174,8 @@
 @@ -223,7 +247,7 @@
  								  Vital="no"
  								  Name="XenServerHealthCheck"
- 								  DisplayName="[Citrix] [XenServer] Health Check Service"
--								  Description="Enables monitoring of the health of your Citrix XenServer pools by periodically uploading data from the servers to Citrix Insight Services."
+ 								  DisplayName="[Citrix XenServer] Health Check Service"
+-								  Description="Enables monitoring of the health of your Citrix Hypervisor pools by periodically uploading data from the servers to Citrix Insight Services."
```

Comment:
We should have [Citrix XenServer] here as well (instead of "Citrix Hypervisor")

Reply:
Good idea.